mpsl/CHANGELOG.rst

```diff
@@ -7,10 +7,10 @@ Changelog
    :local:
    :depth: 2
 
-nRF Connect SDK v2.1.0
-**********************
+Main branch
+***********
 
-All the notable changes included in the |NCS| v2.1.0 release are documented in this section.
+All the notable changes included in the main branch are documented in this section.
```

Comment:
You may want to keep this one as is, or add a new "Main branch" section

Reply:
Added new `Main branch` section, this is no more exactly 2.1.0